common/config.go

```diff
@@ -26,6 +25,22 @@ const (
 var homeDir, _ = os.UserHomeDir()
 var baseDir = path.Join(homeDir, ".atomicswap")
 
+// publicBootnodes are bootnodes with public IP addresses and they used in every
+// environment other than development.
+var publicBootnodes = []string{
```

Comment:
This list combined the former stagenet and mainnet bootnodes. The `bootnode` executable uses port `9909` by default, mainly to reduce conflicts when `swapd` and a `bootnode` are running on the same host. If there were 2 entries from the same IP, I picked the one on port `9909`. Some of the ones on port `9900` are probably `swapd` instances, which is fine too.